locales/en/messages.json

```diff
@@ -109,6 +109,10 @@
         "message": "Show all serial devices (for manufacturers or development)",
         "description": "Do not filter serial devices using VID/PID values (for manufacturers or development)"
     },
+    "showVirtualMode": {
+        "message": "Enable virtual connection mode",
+        "description": "Hide virtual connection mode by default"
```

Comment:
This description is only for the translators. It must "describe" what are they seeing to help them translate them.

Reply:
Fixed!